src/Nethermind/Nethermind.Specs/Forks/16_Cancun.cs

```diff
@@ -17,6 +18,8 @@ protected Cancun()
             IsEip5656Enabled = true;
             IsEip4844Enabled = true;
             IsEip6780Enabled = true;
+            IsEip4788Enabled = true;
+            Eip4788ContractAddress = Address.FromNumber(0x0b);
```

Comment:
This might differ from one network to another (hopefully not)

Reply:
it will